src/api/article/localized/controller.js

```diff
@@ -1,13 +1,26 @@
 import { checkIsFound } from 'utils/validation';
 import { sendJson } from 'utils/api';
+import HttpError from 'node-http-error';
 
 import Article from 'api/article/article.model';
 
 import LocalizedArticle from './model';
 
 export const create = async ({ params: { articleId }, body }, res, next) =>
   Article.findOne({ _id: articleId })
+    .populate('locales', 'locale')
     .then(checkIsFound)
+    .then(article => {
+      article.locales.forEach(loc => {
```

Comment:
```js
if (article.locales.includes(body.locale)) {
	throw new HttpError(...)
}
```

Reply:
it's not the same: `article.locales` here is an array of object, each containing a `locale` field. I can't *include* over it.

Reply:
```js
article.locales.map(({ locale }) => locale).includes(body.locale)